examples/specs/window_transform_average_rating.vl.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+    "description": "Bar graph showing how each film differs from the average rating for that year",
```

Comment:
I think this example is quite confusing.   
I took me a while to get that "Good" "Bad" "Average" are movie names.  Maybe better put actual movie names in there?  

Also, it's kinda weird that you calculate average ratings for one movie in 2015. 

Perhaps, try to re-create examples using the movies data instead? (Feel free to play with it in  Voyager a bit.)

Reply:
I liked this example because it could group by year. I changed the movie names to be Move One etc. I also added another example to use the movies.json file and do something similar there.

Reply:
I'd add another movie for 2015 to make it less weird.   Also, looks like the bar is acting weird here.  (Why isn't it starting from zero?)  -- Use point for now and file a bug?

Reply:
Btw, this example seems redundant with window_transform_movie_mean_difference.vl.json? 

An improved window_transform_movie_mean_difference.vl.json would be way more interesting, so remove this one? 

(Still, please file a bug that bar for this one is wrong.)